css/site.css

```diff
@@ -106,7 +106,7 @@ div.jstree-wholerow-clicked {
 .mscom-footer2 .credits {
     font-size: 12px;
 }
-a.msopentech-link, a.msopentech-link:hover, a.msopentech-link:active {
+a.mmsft-link, a.msft-link:hover, a.msft-link:active {
```

Comment:
Fixed.

From: William Bartholomew [mailto:notifications@github.com]
Sent: Friday, December 11, 2015 12:51 PM
To: Microsoft/microsoft.github.io <microsoft.github.io@noreply.github.com>
Cc: Candice Pfeister (CompuCom Systems Inc) <v-capfei@microsoft.com>
Subject: Re: [microsoft.github.io] Updated site info and some fixes (#124)


In css/site.css<https://github.com/Microsoft/microsoft.github.io/pull/124#discussion_r47403663>:

> @@ -106,7 +106,7 @@ div.jstree-wholerow-clicked {

>  .mscom-footer2 .credits {

>      font-size: 12px;

>  }

> -a.msopentech-link, a.msopentech-link:hover, a.msopentech-link:active {

> +a.mmsft-link, a.msft-link:hover, a.msft-link:active {

Should the first one be msft not mmsft?

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/microsoft.github.io/pull/124/files#r47403663>.